tests/test_AnoGam.py

```diff
@@ -17,35 +17,46 @@ def test_name(self):
     def test_common_name(self):
         assert self.species.common_name == "Anopheles gambiae"
 
-    # QC Tests. These tests are performed by another contributor
-    # independently referring to the citations provided in the
-    # species definition, filling in the appropriate values
-    # and deleting the pytest "skip" annotations.
-    @pytest.mark.skip("Population size QC not done yet")
     def test_qc_population_size(self):
-        assert self.species.population_size == -1
+        # based on theta = 4 Ne u since diversity in most pops is 1.5%
+        assert pytest.approx(self.species.population_size, 0.015 / 3.5e-9)
 
-    @pytest.mark.skip("Generation time QC not done yet")
     def test_qc_generation_time(self):
-        assert self.species.generation_time == -1
+        assert self.species.generation_time == 1 / 11
 
 
 class TestGenomeData(test_species.GenomeTestBase):
 
     genome = stdpopsim.get_species("AnoGam").genome
+    mut_rate = 3.5e-9
+    recomb_rate = 1.4e-8
 
-    @pytest.mark.skip("Recombination rate QC not done yet")
+    # obtained as total lengths of maps at
+    # ftp://ngs.sanger.ac.uk/production/ag1000g/phase1/AR3.1/recombination_maps/
     @pytest.mark.parametrize(
         ["name", "rate"],
-        {"2L": -1, "2R": -1, "3L": -1, "3R": -1, "X": -1, "Mt": -1}.items(),
+        {
```

Comment:
I'm not sure where the recombination rates in the code come from either, other than the 0 rate on 2L? I couldn't find them in the paper. The numbers I got below from the average across the AR3.1-released maps, but they don't match.

Reply:
got the rates from here:

![image](https://user-images.githubusercontent.com/7674197/114593328-d62c4e80-9c40-11eb-831f-e805a314122d.png)

Reply:
hah, right, sorry. was looking at the wrong paper.

Reply:
Hm - but, those numbers are for 3L, 3R, and 2R, and are actually from Zheng et al. Pombi et al report a map for the X but don't give an overall rate. Computing an overall rate from the endpoints of their table 1 gets a rate of about 2cM/Mb, not 1cM/Mb like we've got here. But, they say that the X has possibly more weird stuff than 2L, which you've already set to 0 because of the inversion. (I'm not sure setting to 0 is the right thing, honestly...)  Wouldn't it be better to use endpoints of the genetic map that was released by Ag1000g?

Reply:
i think these are the numbers from this study, not Zheng et al? We can use the ag1000g map though, yes

Reply:
well, it says "based on the data of Zheng and others" - this study only collected data on the X.

Happy to go with whatever you want - maybe could you put in the numbers you like so I can QC them? Either push a commit to this PR or just post them in comments?

Reply:
I gave the Zheng et al. and the Pombi et al papers a second read. Zheng et al. is a linkage map from the pre-genome era, so there are no estimates of physical distance there. What _seems_ to be going on is that Pombi et al, with a genome in hand, are now able to combine the physical and genetic maps to get these numbers in units of cM/Mb. Indeed in doing so they are able to confirm some of the observations from Zheng et al about discordance between polytene band size and local recombination rate on the X. 

So- I think the original numbers in my PR were correct here and that adding the Zheng et al citation is important.

Reply:
Ok, sounds good. And, what to do about X (and, 2L)?

Reply:
We've decided to take the overall rough rate given in Pombi for X (1e-8) and the 2R rate for 2L (1.3.e-8)